File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        n = len(nums)
        ans , res = -1,-1
        for i in range(n):
            if nums[i] == target:
                ans = i
                break
        for i in range(n-1,-1,-1):
            if nums[i] == target:
                res = i
                break
        return (ans,res)

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        int n=nums.size();
        int res =-1, ans = -1;
        for(int i=0;i<n;i++){
            if(nums[i]==target){
                ans = i;
                break;
            }
        }
        for(int i=n-1;i>=0;i--){
            if(nums[i]==target){
                res = i;
                break;
            }
        }
        return {ans,res};
    }
};